chcemvediet/apps/__init__.py

```diff
@@ -0,0 +1,25 @@
+import datetime
```

Comment:
Pls premenuj nazov PR, a teda aj vysledneho commitu po mergnuti, na "#293 Workaround sqlite timezone bug", lebo "Strip timezone suffix" zneje prilis nejasne, ze co to vlastne robi.

(tento komentar nesuvisi s riadkom, ku ktoremu je zaveseny; je tu zaveseny len preto, aby sa dal resolvnut)

Reply:
:ok: